lib/scripts/users_list.rb

```diff
@@ -0,0 +1,32 @@
+require 'csv'
+
+class UsersList
+  attr_accessor :csv_file
+
+  def initialize
+    @csv_file = CSV.open(File.join(Rails.root, "/lib/scripts/results/users_list.csv"), "w") do |csv|
```

Comment:
Instead of opening the CSV on class initialization I would open it and close it in `#populate_user_info`. It seems like it would be easy to instantiate a UsersList, but not run populate_user_info or run it with an error and leave the file open.

Reply:
With the `#open` method, if a block is provided, the CSV object will close as the block terminates. It made sense to me to initialize a csv file so we can add headers here. Still, I think it makes more sense to do the open/close in the `#populate_user_info`, so I will move it there.